compiler/tf2tfliteV2/README.md

```diff
@@ -41,15 +47,15 @@ python tf2tfliteV2.py \
   -h, --help            show this help message and exit
   --v1                  Use TensorFlow Lite Converter 1.x
   --v2                  Use TensorFlow Lite Converter 2.x
-  --input_path INPUT_PATH
+  -i INPUT_PATH, --input_path INPUT_PATH
                         Full filepath of the input file.
-  --output_path OUTPUT_PATH
+  -o OUTPUT_PATH, --output_path OUTPUT_PATH
                         Full filepath of the output file.
-  --input_arrays INPUT_ARRAYS
+  -ia INPUT_ARRAYS, --input_arrays INPUT_ARRAYS
```

Comment:
Usually I've seen single `-` with single char for options and double `--` for long names for the options.
I'm not sure two character is used before. Have you found any that provides like this?
We can use `-I` instead of `-ia` and `-s` instead of `-is` and `-O` instead of `-oa`.